compass/landice/tests/ensemble_generator/ensemble_generator.cfg

```diff
@@ -0,0 +1,21 @@
+# config options for setting up an ensemble
+[ensemble]
+
+# start and end numbers for runs to set up and run
+# Additional runs can be added and run to an existing ensemble
+# without affecting existing runs, but trying to set up a run
+# that already exists may result in unexpected behavior.
+# Run numbers should be zero-based
+# These values do not affect viz/analysis, which will include any
+# runs it finds.
+start_run = 0
+end_run = 3
+
+param_vector_filename = Sobol_Initializations_seed_4_samples_100.csv
+
+input_file_path = /global/cfs/cdirs/fanssie/MALI_projects/Thwaites_UQ/Thwaites_4to20km_r02_20230126/Thwaites_4to20km_r02_20230126.nc
+# Base path to the input files. User has to supply.  This path needs to include the files listed below.  Eventually this could be hard-coded to use files on the input data server, but initially we want flexibility to experiment with different inputs and forcings
```

Comment:
This comment should be above the associated config option for it to translate correctly into the config file in the setup directory.  This comment will get attached to `orig_fric_exp` where it is.